exercises/practice/minima/minima.py

```diff
@@ -0,0 +1,6 @@
+from z3 import *
+
+
+def minima(a: float, b: float, c: float, d: float):
+    """Finds a local minima on the given function."""
```

Comment:
I would add a comment here indicating that this is where the user should add their code (you can use the same one that I have in mine to be consistent if you want).